docs/support-matrix.md

```diff
@@ -0,0 +1,51 @@
+# Compatible Storage Providers
+
+Ark is compatible with a variety of storage providers for backup data, as well as for volume snapshots using either a plugin for the volume provider, or [restic][4] for filesystem level backups.
+
+As of Ark v0.6.0, Ark provides a plugin system that allows anyone to add compatibility for additional backup and volume storage platforms without modifying the Ark codebase.
+
+## Backup Storage Providers
+
+| Provider                  | Owner    | Contact                         |
+|---------------------------|----------|---------------------------------|
+| [AWS S3][2]               | Ark Team | [Slack][10], [GitHub Issue][11] |
+| [Azure Blob Storage][3]   | Ark Team |                                 |
+| [Google Cloud Storage][4] | Ark Team |                                 |
+
+## S3-Compatible Backup Storage Providers
+
+Ark interfaces with the S3 API via [Amazon's Go SDK][12]. Some third party object storage providers use the S3 API, and users have reported the following will work using Ark's S3 support.
```

Comment:
"Ark implements [Amazon's Go SDK][12] to connect to the S3 API. Some third-party storage providers also support the S3 API, and users have reported the following providers work with Ark:"